src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -4,7 +4,10 @@ public class Bulldozer extends Machine {
     private double bladeWidth;
     private double weight;
 
-    public Bulldozer() {
```

Comment:
You shouldn't delete this construtor, I said ADD new construtor, but this one should be present (In all classes that `extends Machine`).